src/__tests__/index.test.js

```diff
@@ -127,3 +127,23 @@ describe('transforming CSS units', () => {
     'p.low { pitch: 105hz; }'
   );
 });
+
+describe('transforming @rules', () => {
+  run(
+    'should safely transform the @media and its properties',
+    '@MEDIA screen and (min-width: 480px){body{COLOR: lightgreen;}}',
+    '@media screen and (min-width: 480px){body{color: lightgreen;}}'
+  );
+
+  run(
+    'should safely transform the @namespace',
+    '@NAMESPACE prefix url(http://www.w3.org/1999/xhtml);',
+    '@namespace prefix url(http://www.w3.org/1999/xhtml);'
+  );
+
+  run(
+    'should safely transform the @media but not the params',
+    '@MEDIA SCREEN and (min-width: 480px){}',
+    '@media SCREEN and (min-width: 480px){}'
```

Comment:
Word like `SCREEN` and etc should be too lowercased

Reply:
I couldnt find a parser to parse to parse them. 
the only way to transform them was simply to lowercase everything after `@MEDIA` but then it wouldn't be safe as it was transforming the property values (480px) as well.

Is it safe to convert all the values to lowercase ?

Reply:
Yes, it is safe too, we should have known list

Reply:
Okay will change it 👍

Reply:
done